microprot/snakemake/Snakefile

```diff
@@ -23,6 +23,11 @@ try:
 except KeyError:
     pass
 
+# create output directories if they don't exist
+for _out_dir_name in ['PDB', 'CM', 'AB', 'log', 'pkg']:
+    _out_dir_path = '%s/%s' % (config['MICROPROT_OUT'], _out_dir_name)
```

Comment:
this is a legitimate concern for general code portability, but it won't be running on WCG machines for this project. only on Barnacle